build.rs

```diff
@@ -33,13 +36,25 @@ fn download_and_unzip(client: &reqwest::blocking::Client, url: &str) {
 
 #[cfg(feature = "build")]
 fn main() {
+    let target = env::var("TARGET").unwrap();
+    let mut config = cpp_build::Config::new();
+
     println!("cargo:rerun-if-changed=./files");
 
+    //only for windows mingw64/gnu tool chain
```

Comment:
Could you add a space in the end of `//`?

Reply:
yep I ll will do in the next commit.